mvpa2/generators/permutation.py

```diff
@@ -39,7 +39,7 @@ class AttributePermutator(Node):
     dataset.
     """
     def __init__(self, attr, count=1, limit=None, assure=False,
-                 strategy='simple', rng=np.random, **kwargs):
+                 strategy='simple', chunk_attr=None, rng=np.random, **kwargs):
```

Comment:
I don't see such a case. But limit ant chunk_attr does different things. If you specify limit, then it will permute labels only within limit and not between (am I right here?) for chunk strategy it is opposite, you will not permute within chunk_attr, but you will swap around chunk labels in original order between chunks. Also it's doing different thing in the code -> that get_chunk_attr function